### DISCUSSION TITLE
Ette.biz??

### DISCUSSION THREAD

**Discussion by RagnarSTS** [Security-Onion-Solutions/securityonion#3180] [Unsupported Versions] [answered]
So while attempting to troubleshoot map issues [https://github.com/Security-Onion-Solutions/securityonion/discussions/2972](url) I noted that the security onion load is attempting to reach out to ette.biz pretty often.  

Anybody know why?

**Comment by joeb1kenobe** [comment, marked as answer]
That is a cent OS mirror site.

On Sun, Feb 28, 2021 at 1:04 AM RagnarSTS <notifications@github.com> wrote:

> So while attempting to troubleshoot map issues
> https://github.com/Security-Onion-Solutions/securityonion/discussions/2972
> <http://url> I noted that the security onion load is attempting to reach
> out to ette.biz pretty often.
>
> Anybody know why?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/3180>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6WG2XZ7VAFGCWRRKSDTBHMIXANCNFSM4YKW3XYQ>
> .
>
-- 
Joe Brown
